plugins/modules/ome_application_certificate.py

```diff
@@ -160,37 +171,60 @@
 import json
 import os
 from ansible_collections.dellemc.openmanage.plugins.module_utils.ome import RestOME, OmeAnsibleModule
-from ansible.module_utils.six.moves.urllib.error import URLError, HTTPError
+from urllib.error import URLError, HTTPError
 from ansible.module_utils.urls import ConnectionError, SSLValidationError
+from ansible.module_utils.compat.version import LooseVersion
+
+OME_INFO = "ApplicationService/Info"
+LOW_OME = "OME version low"
```

Comment:
```suggestion
LOW_OME = "This operation is not supported on OpenManage Enterprise version {version}"
```
{version} replace with current version.

Reply:
Done